cl/opinion_page/urls.py

```diff
@@ -18,6 +19,11 @@
 urlpatterns = [
     # Court pages
     url(r"^court/(?P<pk>[^/]*)/$", court_homepage, name="court_homepage"),
+    url(
+        r"^court/(?P<pk>[^/]*)/new/$",
```

Comment:
Let's tweak this to make it `/new-opinion/`. `/court/tenn/new` feels too much like where you'd go to upload a new court.

Reply:
Hey good news on using `reverse` everywhere, right?

Reply:
√